docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,87 @@
+- [PMM-12544](https://jira.percona.com/browse/PMM-12544) - Added deprecation notices to the PMM documentation DBaaS pages. For a more streamlined and robust database deployment experience, try [Percona Everest](http://per.co.na/pmm-to-everest).
+- [PMM-12549](https://jira.percona.com/browse/PMM-12549) - Added support for the latest MongoDB version. You can now use PMM to monitor MongoDB 7 databases.
+
+
+## Components upgrade
+
+- [PMM-12154](https://jira.percona.com/browse/PMM-12154) - Updated postgres_exporter to version [0.14.0](https://github.com/prometheus-community/postgres_exporter/releases). With this update, we have resolved several performance issues and eliminated the creation of multiple connections per database.
+- [PMM-12223](https://jira.percona.com/browse/PMM-12223) - Clickhouse has been updated to version 23.8.2.7, which optimizes memory and CPU usage to improve system performance.
+
+
+## Bugs Fixed
+
+- [PMM-4712](https://jira.percona.com/browse/PMM-4712) - We have addressed the issue where the [pprof](https://github.com/google/pprof) heap reports for postgres_exporter were missing. 
+- [PMM-12626](https://jira.percona.com/browse/PMM-12626) - Due to the packages being built on an outdated Go version, there was a potential vulnerability. We have updated Go to the latest version to mitigate this risk.
+- [PMM-12658](https://jira.percona.com/browse/PMM-12658) - Corrected a typo in the MongoDB cluster summary dashboard.
+- [PMM-12414](https://jira.percona.com/browse/PMM-12414) - When the Alertmanager configuration included the `group_by` parameter, accessing the `logs.zip` endpoint resulted in an unexpected behavior (502 response).
+- [PMM-12725](https://jira.percona.com/browse/PMM-12725) - Fixed the pagination for QAN.
+- [PMM-11714](https://jira.percona.com/browse/PMM-11714) - Registering a node with the Grafana Admin flag enabled but a non-admin role was failing. This issue has now been resolved.
+- [PMM-12660](https://jira.percona.com/browse/PMM-12660) - Prior to version 2.41.0 of PMM, the endpoint `/v1/management/Agent/List` could deliver database certificates to the PMM UI, allowing an authenticated admin user to view the output of TLS certificates. This posed a security issue since certificates should be consumed by the backend only. We have resolved this issue now.
+- [PMM-12630](https://jira.percona.com/browse/PMM-12630) - Fixed an issue in which the upgrade process for PMM versions lower than or equal to 2.37.1 was stuck in a loop and could not be completed.
+
+## Known issue
+
+- [PMM-12682](https://jira.percona.com/browse/PMM-12682) PMM doesn't show service names within the PMM Dump dataset when the **All Services** option is selected during PMM Dump creation. This issue will be fixed in a future release.
```

Comment:
Status: MERGED
Probably no longer the bug  / defect

Reply:
@catalinaadam I think we can remove this.